docs/CONTRIBUTING.md

```diff
@@ -10,12 +10,8 @@ way to get these dependencies is via [Anaconda](https://www.anaconda.com/downloa
 conda install -c conda-forge protobuf numpy
 ```
 
-During development, it's convenient to install ONNX in development mode (to disable ONNX-ML, set environment variable `ONNX_ML=0`):
+To build ONNX from source please follow the instructions listed [here](https://github.com/onnx/onnx#build-onnx-from-source).
```

Comment:
delete the lines 5 through 11... All of this should be covered in the build instructions.

Reply:
on line 18 you can add - Assuming build succeed in the initial step simply running 
```
pip install -e .
```
from onnx root dir should work.